src/agentscope_runtime/common/skills/alipay/payment.py

```diff
@@ -0,0 +1,614 @@
+
+    result: str = Field(
+        ...,
+        description="包含链接的 markdown 文本，" "你要将文本插入对话内容中。",
+    )
+
+
+class MobileAlipayPayment(Skill[MobilePaymentInput, PaymentOutput]):
+    """
+    手机端支付宝支付组件
+
+    该组件用于创建适合手机端的支付宝支付订单。生成的支付链接可以在手机浏览器中打开，
+    用户可以跳转到支付宝应用完成支付，或者直接在浏览器中进行支付操作。
+
+    主要特点：
+    - 适用于移动网站和移动应用
+    - 支持支付宝应用内支付和浏览器内支付
+    - 使用QUICK_WAP_WAY产品码
+    - 返回可直接使用的支付链接
+
+    输入参数类型：MobilePaymentInput
+    输出参数类型：PaymentOutput
+
+    使用场景：
+    - 移动端网站支付
+    - 手机App内嵌支付
+    """
+
+    name: str = "alipay_mobile_payment"
+    description: str = (
+        "创建一笔支付宝订单，返回带有支付链接的 Markdown 文本，"
+        "该链接在手机浏览器中打开后可跳转到支付宝或直接在浏览器中支付。"
+        "本工具适用于移动网站或移动 App。"
+    )
+
+    async def _arun(
+        self,
+        args: MobilePaymentInput,
+        **kwargs: Any,
+    ) -> PaymentOutput:
+        """
+        创建手机支付宝支付订单
+
+        该方法用于创建适用于手机浏览器的支付宝支付订单。生成的支付链接可以在手机浏览器中
+        打开，用户可以跳转到支付宝应用或直接在浏览器中完成支付。
+
+        Args:
+            args (MobilePaymentInput): 包含支付参数的输入对象
+                - out_trade_no: 商户订单号
+                - order_title: 订单标题
+                - total_amount: 支付金额（元）
+            **kwargs: 额外的关键字参数
+
+        Returns:
+            PaymentOutput: 包含支付链接的Markdown文本输出
+
+        Raises:
+            ValueError: 当配置参数错误时
+            ImportError: 当支付宝SDK不可用时
+            Exception: 当创建订单过程中发生其他错误时
+        """
+        try:
+            # 创建支付宝客户端实例
+            alipay_client = _create_alipay_client()
+
+            # 创建手机网站支付模型并设置参数
+            model = AlipayTradeWapPayModel()
+            model.out_trade_no = args.out_trade_no  # 商户订单号
+            model.total_amount = str(
+                args.total_amount,
+            )  # 支付金额（转换为字符串）
+            model.subject = args.order_title  # 订单标题
+            model.product_code = "QUICK_WAP_WAY"  # 产品码，固定值
+
+            # 使用自定义的扩展参数类
+            extend_params = AgentExtendParams()
+            extend_params.request_channel_source = X_AGENT_CHANNEL
+            model.extend_params = extend_params
+
+            # 创建手机网站支付请求
+            request = AlipayTradeWapPayRequest(biz_model=model)
+
+            # 设置回调地址（如果配置了环境变量）
+            if AP_RETURN_URL:
+                request.return_url = AP_RETURN_URL
+            if AP_NOTIFY_URL:
+                request.notify_url = AP_NOTIFY_URL
+
+            # 执行请求获取支付链接
+            response = alipay_client.page_execute(request, http_method="GET")
+            return PaymentOutput(
+                result=f"支付链接: [点击完成支付]({response})",
+            )
+
+        except (ValueError, ImportError) as e:
+            # 配置或SDK错误，直接抛出
+            logger.error(f"移动支付配置或SDK错误: {str(e)}")
+            raise
+        except Exception as e:
+            # 其他异常，包装后抛出
+            error_msg = f"创建手机支付订单失败: {str(e)}"
+            logger.error(f"移动支付执行异常: {error_msg}")
+            raise RuntimeError(error_msg) from e
+
+
+class WebPageAlipayPayment(Skill[WebPagePaymentInput, PaymentOutput]):
+    """
```

Comment:
If a Chinese docstring is required, please follow the bilingual format below to ensure readability and consistency:
```python
def add(a: int, b: int) -> int:
    """
    功能:
        计算两个整数的和。

    参数(Args):
        a (int): 第一个整数
        b (int): 第二个整数

    返回(Returns):
        int: 两个整数的和

    ---
    Description:
        Calculate the sum of two integers.

    Args:
        a (int): First integer
        b (int): Second integer

    Returns:
        int: Sum of the two integers
    """
    return a + b
```